pkg/limatmpl/embed.go

```diff
@@ -552,11 +555,72 @@ func (tmpl *Template) combineNetworks() {
 	}
 }
 
+// yamlfmt will fail with a buffer overflow while trying to retain line breaks if the line
+// is longer than 64K. We will encode all text files that have a line that comes close.
+// maxLineLength is a constant; it is only a variable for the benefit of the unit tests.
+var maxLineLength = 65000
```

Comment:
I'm not sure if it's a good idea to declare `maxLineLength` as `var` instead of `const`.
For unit tests, we can use [`strings.Repeat`](https://pkg.go.dev/strings#Repeat) to construct a very long string.

```suggestion
const maxLineLength = 65000
```

Reply:
I was using `strings.Repeat` before, but changed it in response to a request from @AkihiroSuda to add a test for `maxLineLength'.

It is not an exported variable, and the comment right above says it is only mutable so unit tests can mock it. Unit tests should use constant data for "expected" results. Otherwise a thinko in the implementation of the function that is being tested can carry over in the test function constructing the test data and cancel out the error in the implementation. Not that it matters in a trivial case like this.


I think it is fine as is.